Modify field Trial flow chart and edits to RC2 Chptr 4 & 5

steelescotr opened the pull request:
docs\gov\chapter 9
Modifies the Field Trial graphic to replace CNTT with Anuket

gkunz commented:
Hi @steelescotr. Thanks! I noticed that new new png file does not have a transparent background, but a black one. I suggest exporting it again with a transparent background.

Also, I think we should absolutely upload the "editable sources" of the figures. Do you have those available, too?

Finally, I don't think we need a copy of the old file. Git will keep a copy for us.

CsatariGergely reviewed (changes requested):
- We do not need field_trials_old.png, it is stored in git history. Please delete it. 
- Please add the editable version of the file with the name field_trails.pptx

steelescotr commented:
I have updated the Background color on field_trials.png, deleted the .old file, and created new .png files for nfvi_transition and vnf_cnf_tranistion. I added the pptx files under artefacts.

gkunz commented:
Hi @steelescotr. Thanks for the update. The background is now white instead of transparent, but that has been the case for the other two figures as well. I don't mind a white background, so that's ok for me.

CsatariGergely commented on doc/gov/chapters/chapter09.rst:
```diff
@@ -210,7 +210,7 @@ Success Indicators
    -  LaaS (e.g. UNH)
 
-3. Engaged vendors successfully configure their VI and run the RC-1 test suite and are able to provide expert feedback
+3. Engaged vendors successfully configure their VI and run the RC-1 or RC-2 test suite and are able to provide expert feedback
```
```suggestion
3. Engaged vendors successfully configure their VI and run the RC1 or RC2 test suite and are able to provide expert feedback
```

steelescotr replied:
Will remove the dashes.

steelescotr commented:
@CsatariGergely Lease review requested change and approve if you are satisfied with the it.

steelescotr commented:
@walterkozlowski Can you please add your review to this PR?

steelescotr commented:
Thanks for the recommendations Georg.....I will update the graphic, and remove the old file.....
As for the editable source, yes I do have that...it's a pptx file. Will add it to the figures directory.


Scot

805-825-1100

From: Georg Kunz ***@***.***>
Sent: Thursday, September 1, 2022 12:07 AM
To: cntt-n/CNTT ***@***.***>
Cc: Scot Steele ***@***.***>; Mention ***@***.***>
Subject: Re: [cntt-n/CNTT] Modify field Trial flow chart to replace CNTT with Anuket (PR #3137)


Hi @steelescotr<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsteelescotr&data=05%7C01%7Cscotsteele%40microsoft.com%7C4a5032d2f1e24e6f122608da8be88406%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637976128016528714%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=kmOQG%2FZgwquh%2BM8npZXkHxM3Aw1GWGgm0rligHbG0k8%3D&reserved=0>. Thanks! I noticed that new new png file does not have a transparent background, but a black one. I suggest exporting it again with a transparent background.

Also, I think we should absolutely upload the "editable sources" of the figures. Do you have those available, too?

Finally, I don't think we need a copy of the old file. Git will keep a copy for us.

-
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcntt-n%2FCNTT%2Fpull%2F3137%23issuecomment-1233838143&data=05%7C01%7Cscotsteele%40microsoft.com%7C4a5032d2f1e24e6f122608da8be88406%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637976128016528714%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=p6w2fYnFj00oLs71IVwVDCEMw06%2B3Ko3dVnvL9ZzPP4%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAXFPFCN7KGBMCLKYGMWGRFLV4BIX5ANCNFSM6AAAAAAQBWLS6M&data=05%7C01%7Cscotsteele%40microsoft.com%7C4a5032d2f1e24e6f122608da8be88406%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637976128016528714%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=oTVwbkMdG%2F5Y9f47V62eDpr6i4Ev7478PO13W21HkRY%3D&reserved=0>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

steelescotr commented:
Modifications made to the Chapter 4 - included all of the workload tests defined in RA2. 
Chapter 5 updated with references to resources.

CsatariGergely commented on doc/gov/chapters/chapter09.rst:
```diff
@@ -288,7 +288,9 @@ Success Indicators
 
-#. Engaged vendors successfully configure their VI and run the RC-1 test suite and are able to provide expert feedback
+
+#. Engaged vendors successfully configure their VI and run the RC1 or RC2 test suite and are able to provide expert feedback
```
```suggestion
#. Engaged vendors successfully configure their Cloud Infrastructure and run the RC1 or RC2 test suite and are able to provide expert feedback
```

CsatariGergely commented on doc/gov/chapters/chapter09.rst:
```diff
@@ -299,16 +301,17 @@ Initiation
+The object is to quantitively and qualitatively assess and evaluate the following Anuket requirements,
+methods, and support processes:
+
+-  RI1 or RI2 Specifications
```
```suggestion
-  RA1 or RA2 Specifications
```

CsatariGergely commented on doc/gov/chapters/chapter09.rst:
```diff
@@ -299,16 +301,17 @@ Initiation
+methods, and support processes:
+
+-  RI1 or RI2 Specifications
+-  VI implementation support methods ( i.e. cookbooks, installation manuals, how to guides etc.)
```
```suggestion
-  Cloud Infrastructure implementation support methods ( i.e. cookbooks, installation manuals, how to guides etc.)
```

CsatariGergely commented on doc/gov/chapters/chapter09.rst:
```diff
@@ -420,13 +427,15 @@ the methods used by Anuket accordingly to quantitatively and or qualitatively me
-* Benchmark Data
-* Other?
+
+-  VI Implementation methods and procedures (cookbook, etc)
```
```suggestion
-  Cloud Infrastructure Implementation methods and procedures (cookbook, etc)
```

CsatariGergely commented on doc/gov/chapters/chapter09.rst:
```diff
@@ -420,13 +427,15 @@ the methods used by Anuket accordingly to quantitatively and or qualitatively me
-* Other?
+
+-  VI Implementation methods and procedures (cookbook, etc)
+-  RI1 or RI2 Specifications
```
```suggestion
-  RA1 or RA2 Specifications
```

walterkozlowski reviewed (approved):
There are still open change proposals from @CsatariGergely which make sense to me.  I am approving in understanding that these will be taken care of.

steelescotr commented:
@CsatariGergely @walterkozlowski, I have addressed the requested changes. please let me know if there are other issues to resolve.

steelescotr commented:
@rgstori , @CsatariGergely , @walterkozlowski - Could one of you Merge this this PR? it has your approvals already.

collivier commented:
The main RC2 contributor is missing in the review.
And there are many RST issues in it.

I'm very surprised that this patch has not been brought to TSC yet @CsatariGergely.
I think all patchset must be treated fair.
Thank you in advance to apply the same RA1 and RM treatment everywhere.

CsatariGergely commented:
Can you share what rst isses you found in here? I fixed what I could found here: #3289 . Are there any others need fix?

I treat the consumability of Anuket as a priority item as without a decent landing page and documentation, misleading repo names and readthedocs url-s  we can not expect adoption of the project. All of these topics need changes in the "no mans land" not covered in sub-project meetings or projects. This is the reason why I put them to the TSC meeting agenda. If you think other pr-s should be discussed also in TSC, the agenda is open.
